components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIProviderHostObject.java

```diff
@@ -3307,10 +3309,10 @@ public static NativeArray jsFunction_getProviderAPIVersionUsage(Context cx, Scri
         String providerName = (String) args[0];
         String apiName = (String) args[1];
         try {
-            APIUsageStatisticsClient client = new APIUsageStatisticsClient(((APIProviderHostObject) thisObj).getUsername());
+            APIUsageStatisticsRdbmsClientImpl client = new APIUsageStatisticsRdbmsClientImpl(((APIProviderHostObject) thisObj).getUsername());
```

Comment:
removed methods from hostobjects in commit https://github.com/wso2/carbon-apimgt/commit/fd35737f6a9fb080e69b6e915d9ed0f1eee2b6df